src/main/java/com/faforever/client/preferences/TimeInfo.java

```diff
@@ -0,0 +1,25 @@
+package com.faforever.client.preferences;
+
+import java.util.Locale;
+
+public enum TimeInfo {
+  AUTO("System", null), MILLITARY_TIME("24h", new Locale("de", "DE")), UK_TIME("12h", new Locale("en", "UK"));
+  private final Locale usedLocale;
+  private final String dispayName;
```

Comment:
typo
Also, we don't want to hardcode any strings displayed to the user. Store a `displayNameKey` instead, and specify it in the `messages.properties`.